pyamg/strength.py

```diff
@@ -109,12 +109,10 @@ def distance_strength_of_connection(A, V, theta=2.0, relative_drop=True):
     return C
 
 
-def classical_strength_of_connection(A, theta=0.0, norm='abs'):
-    """Classical Strength Measure.
-
+def classical_strength_of_connection(A, theta=0.1, block=None, norm='abs'):
```

Comment:
I would leave the theta=0.0 default and not change that.  That could affect a lot of other unit tests and examples.

Reply:
true, it's just not a good choice of SOC tolerance though. almost never have I found theta=0 to be the "right" choice, which makes me unhappy making it default. @lukeolson ?

Reply:
I agree that a general SOC tol shouldn't be 0, but changing defaults will affect users and affect our unit tests. 
I would suggest having a separate PR to update defaults, and then advertise that or tag it as a major release, so users know what's going on.  I know that PyAMG doesn't have a user base (in any way) like hypre, but this is what hypre had to do every time it changed a default.

Reply:
@lukeolson What say you Luke?  Should we revert theta to 0.0, or use the more practical 0.1 (and make any needed unit test updates to reflect the new default)?